flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/DecimalTypeTest.scala

```diff
@@ -385,11 +385,11 @@ class DecimalTypeTest extends ExpressionTestBase {
 
     testAllApis((-'f51).round(1), "round(-f51,1)", "-100.0")
 
-    testAllApis(('f51).round(-1), "round(f51,-1)", "100")
```

Comment:
The parentheses in the expression here may be redundant, as they do not change the calculation priority and are equivalent to the expression below. By the way, I removed the brackets in the expression. Do we need this change?
`'f51.round(-1) is same as '(f51).round(-1)`.

Reply:
I'm fine with removing it or not. I'm just wondering would it be more clear if don't remove it?

Reply:
Thanks yuxia.
I think it would be better to remove the parentheses. There are also similar code snippets without parentheses in the code above.

```java
    testAllApis('f50.round(-1), "round(f50,-1)", "650")
    testAllApis('f50.round(-2), "round(f50,-2)", "600")
    testAllApis('f50.round(-3), "round(f50,-3)", "1000")
    testAllApis('f50.round(-4), "round(f50,-4)", "0")
```